comp/core/workloadmeta/collectors/catalog-less/options.go

```diff
@@ -40,7 +39,6 @@ func getCollectorOptions() []fx.Option {
 		podman.GetFxOptions(),
 		remoteworkloadmeta.GetFxOptions(),
 		fx.Supply(remoteworkloadmeta.Params{}),
-		processcollector.GetFxOptions(),
```

Comment:
I removed this from the dogstatsd catalog as it imports `github.com/DataDog/datadog-agent/comp/core/workloadmeta/collectors/internal/process` for `process.LocalProcessCollectorIsEnabled()`  causing a large increase in package size. I thought this would be the easiest way forward as the remote collector isn't used in the dogstatsd build. If necessary there should be an alternative such as copying the `LocalProcessCollectorIsEnabled()` code.